### PR TITLE
niebezpiecznik.pl

### PR DESCRIPTION
**Pull request by gork7777**
issue: #105421

### CODE REVIEW

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -6158,6 +6158,8 @@ roweroweporady.pl,tulodz.pl#@#.ad_container
 !+ PLATFORM(ios, ext_android_cb)
 @@||lib.wtg-ads.com/prebid/prebid_*.js$domain=tvn.pl
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/105421
+||niebezpiecznik.pl/wp-content/uploads/*/600x500.jpg
```
Comment (zloyden) [inline EnglishFilter/sections/foreign.txt:6162]: I don't see it. Have you had it on your end?

Comment (gork7777) [reply EnglishFilter/sections/foreign.txt:6162]: Banner is changed I guess. https://imgur.com/a/nuut43i

Comment (gork7777) [reply EnglishFilter/sections/foreign.txt:6162]: Banner is changed I guess. https://imgur.com/a/nuut43i

Comment (zloyden) [reply EnglishFilter/sections/foreign.txt:6162]: Now it's a self-promo, added rules directly.

**Pull request closed without merging**